metagpt/provider/openai_api.py

```diff
@@ -224,13 +224,17 @@ async def acompletion_text(self, messages: list[dict], stream=False) -> str:
         return self.get_choice_text(rsp)
 
     def _calc_usage(self, messages: list[dict], rsp: str) -> dict:
-        usage = {}
         if CONFIG.calc_usage:
-            prompt_tokens = count_message_tokens(messages, self.model)
-            completion_tokens = count_string_tokens(rsp, self.model)
-            usage['prompt_tokens'] = prompt_tokens
-            usage['completion_tokens'] = completion_tokens
-        return usage
+            usage = {}
```

Comment:
Please place this statement before the "if" check and then return it as default.